Utility/treeHelper.FCMacro

```diff
@@ -0,0 +1,338 @@
+__License__ = "MIT"
+__Web__ = "https://github.com/iinjdpa/TreeHelper"
+__Wiki__ = ""
+__Icon__ = "/treeHelper/treeHelper.svg"
+__Help__ = "Select a folder, part or body where helper tree must grow. Drag object from main tree to helper tree"
+__Status__ = "Alpha"
+__Requires__ = "Freecad >= v0.19"
+__Communication__ = "https://github.com/iinjdpa/TreeHelper/issues"
+__Files__ = "/treeHelper/ui_treewindow.py,/treeHelper/treeHelper.svg"
+
+
+import FreeCAD as App
+import FreeCADGui as Gui
+from PySide import QtCore, QtGui
+from PySide.QtCore import Qt, QByteArray, QDataStream, QIODevice
+from PySide.QtGui import QDialog, QApplication, QTreeWidgetItem, QDockWidget, QGraphicsDropShadowEffect, QColor,QMainWindow
+from PySide.QtGui import QMessageBox, QAbstractItemView, QDrag
+import PySide2
+import os
+import sys
+from treeHelper import ui_treewindow
+from collections import OrderedDict
+import time
+
+
+class Objdoc(object):
+    def __init__(self,doc):
+        self.ViewObject = mViewObject()
+        self.Name = doc.Name
+        self.Label = doc.Name
+
+class mViewObject(object):
+    def __init__(self):
+        pass
+    def claimChildren(self):
+        # Nombre del documento
+        doc = App.activeDocument()
+        if doc == None:
+            doc = FreeCAD.newDocument()
+        docName = doc.Name
+        # Buscamos el árbol de FreeCAD. Es la forma de poder importar todos los objetos por orden
+        mw = Gui.getMainWindow()
+        self.tw = mw.findChildren(QtGui.QTreeWidget)
+        self.tw0 = self.tw[0]
+        self.tw0.repaint()
+        self.FCtreeitems = self.tw0.findItems('*', Qt.MatchWrap | Qt.MatchWildcard  | Qt.MatchRecursive)
+        root_labels = []
+        for it in self.FCtreeitems:
+            if not it.parent(): continue
+            if isinstance(it.parent(), list):
+                for itp in it.parent():
+                    if itp == docName:
+                        root_labels.append(it.text(0))
+            else:
+                if it.parent().text(0) == docName:
+                    root_labels.append(it.text(0))
+        root_objects = App.ActiveDocument.RootObjects
+        root_objects_from_labels = []
+        if len(root_labels) == len(root_objects):
+            for obj in root_labels:
+                obj_by_label = App.ActiveDocument.getObjectsByLabel(obj)
+                if len(obj_by_label) > 1:
+                    QMessageBox.critical(self, "Error","There are objects with same label!")
+                    return []
+                else:
+                    root_objects_from_labels.append(App.ActiveDocument.getObjectsByLabel(obj)[0])
+            return root_objects_from_labels
+        else:
+            return root_objects
+
+    def update(self):
+        pass
+
+class TreeDrops(QtGui.QTreeWidget):
+    itemDropped = QtCore.Signal()
+    customMimeType = "application/x-qabstractitemmodeldatalist"
+    def __init__(self, parent=None):
+        super(TreeDrops, self).__init__(parent=None)

```

Comment:
Prefer `super().__init__(parent=None)`. I guess that `super().__init__(parent)` is the correct call anyway.